### ISSUE TITLE
UI on Settings screen: Improve on font size and font color

### ISSUE THREAD

**Issue by archanajjoshi** [mcgillij/AQ#2]
UI: Yellow on white and light grey is not great for visual ease for reading. Stick to black and larger font size text here. Arrows as well, use a black bold arrow. The text `Match 3 or more!` however is OK in yellow but make it bigger for better visibility(not bold, just increased the font size). 

UX part of it is great howevs.

![image](https://github.com/user-attachments/assets/428fe1ce-6e86-4d59-a33b-8e3f3a6cd6c4)

**Comment by mcgillij** [comment]
Yeah I can generate a better screenshot, since that's all this is, I don't even think the UI is the same as in game, since it was taken very early in the dev process. This page will eventually be removed from the "v2" game anyways or at least tied into the first ones.

Thanks for making the issues!

**Comment by mcgillij** [comment]
I removed this screen completely and added a tutorial instead
